### ISSUE TITLE
improve formatting of CSS links

### ISSUE THREAD

**Issue by domurtag** [bertramdev/asset-pipeline#217]
In dev mode, the `<link>` tags that are generated for CSS files are all output on the same line, e.g.

````html
<link rel="stylesheet" href="/foo/assets/jquery-ui-theme.css?compile=false"  /><link rel="stylesheet" href="/foo/assets/common.css?compile=false"  /><link rel="stylesheet" href="/foo/assets/confirm.css?compile=false"  /><link rel="stylesheet" href="/foo/assets/qTip/jquery.qtip.css?compile=false"  /><link rel="stylesheet" href="/foo/assets/qTip/qTipCustomStyles.css?compile=false"  /><link rel="stylesheet" href="/foo/assets/manifests/baseLayout.css?compile=false"  />
````
It would make it easier to see which files are included if each one was output on a separate line, e.g.

````html
<link rel="stylesheet" href="/foo/assets/jquery-ui-theme.css?compile=false"  />
<link rel="stylesheet" href="/foo/assets/common.css?compile=false"  />
<link rel="stylesheet" href="/foo/assets/confirm.css?compile=false"  />
<link rel="stylesheet" href="/foo/assets/qTip/jquery.qtip.css?compile=false"  />
<link rel="stylesheet" href="/foo/assets/qTip/qTipCustomStyles.css?compile=false"  />
<link rel="stylesheet" href="/foo/assets/manifests/baseLayout.css?compile=false"  />
````

**Comment by davydotcom** [comment]
Agreed

**Comment by domurtag** [comment]
It looks like the fix should be as simple as replacing [this line](https://github.com/bertramdev/asset-pipeline/blob/master/grails-app/taglib/asset/pipeline/grails/AssetsTagLib.groovy#L86)

    out << "<link rel=\"stylesheet\" href=\"${depAssetPath}?${modifierParams.join("&")}\" ${paramsToHtmlAttr(attrs)} />"

with 

    out << "<link rel=\"stylesheet\" href=\"${depAssetPath}?${modifierParams.join("&")}\" ${paramsToHtmlAttr(attrs)} />${System.getProperty('line.separator')}"

But the call to `System.getProperty('line.separator')` should only be done once. I'll submit a PR if someone doesn't get there before me.

**Comment by domurtag** [comment]
Thanks, I've left [a comment](https://github.com/bertramdev/asset-pipeline/commit/be8b1d1680f353b080212a1dfd8830536d054abc#comments) suggesting an improvement to your fix.

**Comment by davydotcom** [comment]
Fixed in 2.1.1
